README.md

```diff
@@ -21,3 +21,15 @@ $ make
 
 [nasm]: https://www.nasm.us/
 [qemu]: https://www.qemu.org/
+
+## Building on MacOS
```

Comment:
Please make this a subsection of the "Dependencies" section. It's about installing dependencies anyway.